File: 3_strings/2174.py
# -*- coding: utf-8 -*-
# https://www.urionlinejudge.com.br/judge/en/problems/view/2174

N = int(input())
S = set()

for i in range(N):
    S.add(input())

print("Falta(m) %d pomekon(s)." % (151 - len(S)))

File: 3_strings/2174.cpp
// https://www.urionlinejudge.com.br/judge/en/problems/view/2174

#include <iostream>
#include <string>
#include <set>

int main()
{
    int N;
    std::set<std::string> S;
    std::string pomekon;

    std::cin >> N;

    while (N) {
        std::cin >> pomekon;
        S.insert(pomekon);
        --N;
    }

    std::cout << "Falta(m) " << 151 - S.size() << " pomekon(s).\n";

    return 0;
}
